cardano-diffusion/lib/Cardano/Network/PeerSelection/Governor/Types.hs

```diff
@@ -240,7 +240,10 @@ cardanoPeerSelectionGovernorArgs extraActions =
           <> monitorLedgerStateJudgement psa pst
           <> waitForSystemToQuiesce          pst
       , postBlocking    = mempty
-      , postNonBlocking = mempty
+      , postNonBlocking = \_psp _psa pst ->
+          -- ensure this is run in case when peer selection has nothing else
+          -- that can wake the governor up.
+          GuardedSkip $ Cardano.bootstrapPeersTimeout (extraState pst)
```

Comment:
It would be better to add this to the guarded decision which actually needs to run.  Isn't it this one `waitForSystemToQuiesce`?

Do we have a counter example to test it?